### DISCUSSION TITLE
Extracting motif matches matrix with genomic coordinates

### DISCUSSION THREAD

**Discussion by fwim** [GreenleafLab/ArchR#1003] [Questions / Documentation] [answered]
Dear ArchR Team,
 
I started using ArchR a couple of months back for a large scATAC-seq experiment and I am so excited about how well it runs on a standard macbook pro.
 
Now, I am trying to extract a motif matches matrix (TF motifs as cols, peaks as row, cells with binary data t/f for presence/absence of TF binding motif in peak) for downstream analysis. I used “getMatches()” but there is no information on the peak coordinates (chr, start, end) in the row annotations or row names of the object returned by the function.
 
Do you know whether there is a way to retrieve a motif matches matrix that contains the genomic coordinates of the peaks?
 
 
Many thanks,
Florian

**Comment by rcorces** [comment, marked as answer]
The peak order in the return of `getMatches()` is the same as the peak order from `getPeakSet()`.
I have updated the function description via https://github.com/GreenleafLab/ArchR/commit/18bcead479721978f1d12c21931d83fe294b56d7

**Reply by fwim** [reply]
Many thanks! That solved the issue.
